tests/integration/test_backup_restore_on_cluster/test_disallow_concurrency.py

```diff
@@ -125,19 +125,29 @@ def test_concurrent_backups_on_same_node():
         .query(f"BACKUP TABLE tbl ON CLUSTER 'cluster' TO {backup_name} ASYNC")
         .split("\t")[0]
     )
-    assert_eq_with_retry(
-        nodes[0],
-        f"SELECT status FROM system.backups WHERE status == 'CREATING_BACKUP' AND id = '{id}'",
-        "CREATING_BACKUP",
+
+    status = (
+        nodes[0]
+        .query(f"SELECT status FROM system.backups WHERE id == '{id}'")
+        .rstrip("\n")
     )
-    assert "Concurrent backups not supported" in nodes[0].query_and_get_error(
```

Comment:
Here was the second problem - if the first backup is already created we won't see the error `Concurrent backups not supported`, we'll see the error `Backup already exists`.